### ISSUE TITLE
Unable to create new users

### ISSUE THREAD

**Issue by measeaustin** [JakeHam/TheVirtualCloset#24]
Unable to create new users, get "Creation of user unsuccessful! Try again!" regardless of input

**Comment by parcelt** [comment]
As for this issue, on my initial first run of the application, while running ionic serve, I was able to create a new user. Then strangely enough, I was not able to create any new users afterwards. 
Next, I thought that maybe the issue was because I was still in my account and need to be logged out of it in order to create another new user, but to no avail after logging out as processed by the app, I was not able to create another new user.

**Comment by chensonghu2013** [comment]
I agree with parcelt but the thing is there is no option for us to log out. After logging out, my account is still activated and the "Creation of user unsuccessful! Try again!" still existing
